pkg/aggregator/aggregator.go

```diff
@@ -354,11 +358,46 @@ func NewBufferedAggregator(s serializer.MetricSerializer, eventPlatformForwarder
 		flushAndSerializeInParallel: NewFlushAndSerializeInParallel(pkgconfigsetup.Datadog()),
 
 		filterListChan: make(chan utilstrings.Matcher),
+		tagFilterList:  loadTagFilterList(),
 	}
 
 	return aggregator
 }
 
+func loadTagFilterList() *filterlist.TagMatcher {
+	tagFilterListInterface := pkgconfigsetup.Datadog().GetStringMap("metric_tag_filterlist")
+
+	tagFilterList := make(map[string]filterlist.MetricTagList, len(tagFilterListInterface))
+	for metricName, tags := range tagFilterListInterface {
+		// Tags can be configured as a simple array of tags - default to exclude.
+		arr, ok := tags.([]string)
+		if ok {
+			tagFilterList[metricName] = filterlist.MetricTagList{
+				Tags:   arr,
+				Action: "exclude",
+			}
+		} else {
+			// Or tags can be configured as an object with fields:
+			// tags - array of tags
+			// action - either `include` or `exclude`.
+			// Roundtrip the struct through yaml to load it.
+			tagBytes, err := yaml.Marshal(tags)
+			if err != nil {
+				log.Errorf("invalid configuration for `metric_tag_filterlist` %s", metricName)
+			} else {
+				var tags filterlist.MetricTagList
+				err = yaml.Unmarshal(tagBytes, &tags)
+				if err != nil {
+					log.Errorf("error loading configuration for `metric_tag_filterlist` %s", err)
```

Comment:
And include the faulty key here in addition to the error:
```suggestion
					log.Errorf("error loading configuration from %q: %s", "metric_tag_filterlist ."+metricName, err)
```